app/assets/v2/js/pages/join_tribe.js

```diff
@@ -83,3 +83,59 @@ const tribeLeader = () => {
 };
 
 tribeLeader();
+
+const newManageTribe = () => {
+  $('[data-tribe]').each(function(index, elem) {
+    $(elem).mouseenter(function(e) {
+      if ($(elem).hasClass('btn-outline-gc-green')) {
+        $(elem).addClass('btn-gc-outline-red').text('Unfollow');
+        $(elem).removeClass('btn-outline-gc-green');
+      }
+    }).mouseleave(function(e) {
+      if ($(elem).hasClass('btn-gc-outline-red')) {
+        $(elem).removeClass('btn-gc-outline-red');
+        $(elem).addClass('btn-outline-gc-green').text('Following');
+      }
+    });
+
+    $(elem).focus(function(e) {
+      if ($(elem).hasClass('btn-outline-gc-green')) {
+        $(elem).addClass('btn-gc-outline-red').text('Unfollow');
+        $(elem).removeClass('btn-outline-gc-green');
+      }
+    }).focusout(function(e) {
+      if ($(elem).hasClass('btn-gc-outline-red')) {
+        $(elem).removeClass('btn-gc-outline-red');
+        $(elem).addClass('btn-outline-gc-green').text('Following');
+      }
+    });
```

Comment:
please do the visual mouse changes with css instead and just change the button on click